agent/nmap_agent.py

```diff
@@ -283,12 +283,19 @@ def _emit_network_scan_finding(
     def _emit_services(
         self, scan_results: Dict[str, Any], domain_name: Optional[str]
     ) -> None:
-        logger.info("sending results to %s", domain_name)
-        if (
-            scan_results is not None
-            and scan_results.get("nmaprun") is not None
-            and scan_results["nmaprun"].get("host") is not None
-        ):
+        logger.info("Services targeting domain `%s`.", domain_name)
+        if scan_results is not None and scan_results.get("nmaprun") is not None:
```

Comment:
If you are checking `scan_results is not None`, then there is an issue with the typing of this argument no ?

Reply:
Also, would look cleaner if the condition is flipped and lowered the indentation level of the code below, don't u think?

Reply:
I don't understand your comment. Can you please clarify? I am merging but if something needs to be changed, will apply it.